packages/metal-soy/test/Soy.js

```diff
@@ -11,6 +11,7 @@ import { Events as EventsComponent } from './assets/Events.soy.js';
 // on it.
 // TODO: We should have a better dependency management for soy files so that
 // the order in which they're required doesn't matter.
+import { ComputedData as ComputedDataComponent } from './assets/ComputedData.soy.js';
```

Comment:
Yes, that's correct, I've just realized that this TODO isn't necessary anymore, all imports can already be in alphabetical order today :)